.env.example

```
@@ -1,32 +1,54 @@
-# 환경 설정
-ENVIRONMENT=development
-DEBUG=true
+# ===========================================
+# 기본 배포 설정
+# ===========================================
+RESOURCE_GROUP=
+LOCATION=Korea Central
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The LOCATION key should go before the RESOURCE_GROUP key

(UnorderedKey)